style.css

```diff
@@ -631,6 +631,12 @@ header.hideNav {
 
 /******************************  Pop Up  Ovelay  Endss  ****************************/
 
+.validation {
+  color: #f8f8f8;
+  font-size: 1rem;
+  letter-spacing: 1px;
+}
+
```

Comment:
## YOU DID A GOOD JOB 👏

- [ ] [OPTIONAL] I have one suggestion for you to help you make it better 👍. Please kindly increate validation error message` font size` to around `14-16px` and make its `color` red. Good luck 🍀